docs/nodes/credentials/Twake/README.md

```diff
@@ -0,0 +1,21 @@
+# Twake
+
+You can find information about the operations supported by the Twake node on the [integrations](https://n8n.io/integrations/n8n-nodes-base.twake) page. You can also browse the source code of the node on [GitHub](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Twake).
+
+## Prerequisites
+
+Create a [Twake](https://twake.app/) account.
+
+## Using Access Token
+
+1. Access your [Twake](https://web.twake.app) workspace.
+2. Click on 'Main' in the top left corner.
+3. Select 'Workspace settings' from the dropdown list.
+4. Select 'Applications and connectors'
+5. Click on 'Search applications...'.
+6. Search for 'n8n' and click on 'Display'.
+7. Click on 'Install' and select 'Confirm'.
+8. Click on 'Configure' and copy the Workspace key.
```

Comment:
'Workspace key' -> Is that the correct capitalization?

Reply:
That is how they have capitalized in the Twake app.

Reply:
Gotcha!